validator/validators/stress/stress_validator.go

```diff
@@ -122,6 +142,16 @@ var (
 				"net_bytes_sent":       float64(6820000),
 				"net_packets_sent":     float64(8300),
 			},
+			"emf": {
```

Comment:
Add a upper bound for stress, in this way, if the plugin `emf` being sent X amount of metrics per minute and the CWA cross the upper bound, it would return an errors (only support 1000, 5000, 10000, 50000)

Reply:
The validator already [validates the fix string received](https://github.com/aws/amazon-cloudwatch-agent-test/pull/174/files#diff-797a68b028fdf868b4b0cba0f575d9fad8abcb4f2a4fb6df143d4c6beadf256dR18)